src/api/subscriptions.js

```diff
@@ -124,9 +130,15 @@ export class Subscriptions {
 
   /**
    * Starts subscription purchase flow.
-   * @param {string|SubscriptionRequest} skuOrSubscriptionRequest
+   * @param {string} sku
+   */
+  subscribe(sku) {}
```

Comment:
Why subscribe() API is being changed?
This API is already being used and could break existing implementation.

Reply:
Currently since update subscription is still behind an experimental flag, publishers should only have been passing in a string anyway (or else they would have encountered an error telling them the feature is not yet launched). This change just enforces subscribe taking only a string - and therefore only being used for a new subscription - and then updateSubscription will take the a SubscriptionRequest object.

Reply:
+1. This is fine and shouldn't break anything (the "SubscriptionRequest" was added by me but never launched)